ui/component/liquidateSupports/view.jsx

```diff
@@ -0,0 +1,133 @@
+// @flow
+import * as ICONS from 'constants/icons';
+import React, { useEffect, useState } from 'react';
+import CreditAmount from 'component/common/credit-amount';
+import Button from 'component/button';
+import Icon from 'component/common/icon';
+import { Form, FormField } from 'component/common/form';
+import Card from 'component/common/card';
+
+type Props = {
+  balance: number,
+  totalBalance: number,
+  claimsBalance: number,
+  supportsBalance: number,
+  tipsBalance: number,
+  claim: any,
+  metaData: any,
+  handleClose: () => void,
+  abandonSupportForClaim: (string, boolean | string, boolean) => any,
+};
+
+const LiquidateSupports = (props: Props) => {
+  const { claim, abandonSupportForClaim, handleClose } = props;
+  const [previewBalance, setPreviewBalance] = useState(1);
+  const [amount, setAmount] = useState(0);
+  const [error, setError] = useState(false);
+  const [message, setMessage] = useState('Amount to unlock');
+  const keep =
+    Number(amount) < previewBalance ? Number.parseFloat(String(previewBalance - Number(amount))).toFixed(8) : false;
+  const claimId = claim && claim.claim_id;
+
+  const MESSAGE_NUMBER = __('Amount must be a number');
+  const MESSAGE_BLANK = __('Amount cannot be blank');
+  const MESSAGE_STAKED = __('Your content will do better with more staked on it');
+  const MESSAGE_MAX = __('Amount cannot be more than available');
+  const MESSAGE_ZERO = __('Amount cannot be zero');
```

Comment:
This should be an error

Reply:
at least some of this error detection and handling looks like it should move into an `<InputNumber max={foo} min={bar} />` style design, at least eventually

Reply:
I didn't change this error / messaging situation.